src/components/global/ClassroomModalContent.tsx

```diff
@@ -0,0 +1,37 @@
+import Dialog from '@mui/material/Dialog'
+import DialogContent from '@mui/material/DialogContent'
+import DialogTitle from '@mui/material/DialogTitle'
+import TextField from '@mui/material/TextField'
+import * as React from 'react'
+import Grid from '@mui/material/Grid'
+
+interface Props {
```

Comment:
Great job figuring out how to type your component props! Just note that you can also destructure your props object and then type your props inline with the function declaration, instead of creating a separate interface. See other examples in the codebase. The method you choose is up to personal preference